Frontend/src/components/AdminSettings/UserProfileContainer.css

```diff
@@ -0,0 +1,27 @@
+.container {
```

Comment:
We're going to change this className to 'userProfileContainer', "container" will spill its designs into all other "container"s